models/Articles.model.js

```diff
@@ -0,0 +1,10 @@
+const db = require("../db/connection")
+
+exports.updateVotes = (id, updateValue)=>{
+    return db.query(`UPDATE articles
+        SET votes = votes + ${updateValue}
+        WHERE article_id = ${id}
+        RETURNING *`).then((article)=>{
+            if(article.rowCount === 0){throw article}
```

Comment:
throwing doesn't give you as much flexibility with error handling that a Promise.reject would in this situation. Consider having a look into the notes on Promise.reject to see what you could do in this instance instead.